packages/dialtone-vue2/components/avatar/avatar.vue

```diff
@@ -416,8 +412,8 @@ export default {
     },
 
     validateProps () {
-      if (this.imageSrc && !(this.fullName || this.imageAlt)) {
-        throw new Error('full-name or image-alt must be set if image-src is provided');
+      if (this.imageSrc && !this.imageAlt) {
+        throw new Error('image-alt must be set if image-src is provided');
```

Comment:
Is this actually what we want? The case we're trying to fix here is that the image-alt is repeated as text, like in contact row: https://dialtone.dialpad.com/vue/index.html?path=/story/recipes-leftbar-contact-row--default 

It already states the name in text, meaning it should have no image-alt even when image-src is provided.

Reply:
Hmm, I cannot reproduce the issue, maybe I'm misunderstanding what the issue is here.

Previously we were defaulting the avatar image-alt to the name, so I removed that. Now, you need to provide an image alt that should actually be different from the name, right? as you're describing the image, not the actual name of the contact.

Reply:
It should just have no image-alt at all if it's already described by the name text.